Add ADS Select documentation (#34714)
home-assistant/home-assistant.github.io: source/_integrations/ads.markdown

  description: Sets the [class of the device](/integrations/cover/), changing the device state and icon that is displayed on the frontend.
  type: device_class
{% endconfiguration %}

## Select

> The `ads` select platform accesses an ENUM (int) variable on the connected ADS device. The variable is identified by its name. You have to set up a corresponding ENUM in the TwinCAT PLC, and recommended to use explicit values starting from `0`.

Reviewer: ```suggestion
The `ads` select entity accesses an ENUM (int) variable on the connected ADS device. The variable is identified by its name. You have to set up a corresponding ENUM in the TwinCAT PLC. It is recommended to use explicit values starting from `0`.
```
